### DISCUSSION TITLE
does it make root section like "docs"

### DISCUSSION THREAD

**Discussion by iju707** [imfing/hextra#81] [Q&A] [answered]
I want to make section bellow.

content
  - blog
  - test1
  - test2

blog is used "blog layout"
but test1, test2 are used "_default layout", but I want to use "docs layout"

and I find a solution, copying docs layout files into layout/test1 or layout/test2.
but if hextra is upgraded, I copy again. It's too cumbersome.

**Comment by imfing** [comment, marked as answer]
Can you try this: https://github.com/imfing/hextra/discussions/18

**Comment by iju707** [comment]
Thanks! It works.
